packages/starterkit-twig-demo/patternlab-config.json

```diff
@@ -45,7 +45,7 @@
           "id": "macros",
           "recursive": true,
           "paths": [
-            "source/_macros"
+            "source/_patterns/macros"
```

Comment:
I think this change is also required here https://github.com/pattern-lab/patternlab-node/blob/fd5db84e2caa77472754111611fa84cd45b09df9/packages/development-edition-engine-twig/patternlab-config.json#L117

Reply:
I did: https://github.com/pattern-lab/patternlab-node/pull/1490/files#diff-3b0388a7ba216535ac75d31a9f5afbc1ac71ecdf0aa10bccfc7c8edb6d8a5896